kofta/src/app/components/LanguageSelector.tsx

```diff
@@ -65,6 +65,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "tr", label: "🇹🇷 Türkçe" }, // Turkish
     { value: "uk", label: "🇺🇦 Українська" }, // Ukrainian
     { value: "ur", label: "🇵🇰 اردو" }, // Urdu
+    { value: "uz", label: "🇺🇿 Oʻzbek" }, // Uzbek
```

Comment:
add it to kibbeh's lang selector aswell

Reply:
` { value: "uz", label: "🇺🇿 Oʻzbek" }, // Uzbek` line was already added to kibbeh/src/ui/LanguageSelector.tsx as well.
Please let me know if I should add anything else. Thanks!

Reply:
oh my bad i didnt notice. looks good